backend/tests/environments/test_router.py

```diff
@@ -42,3 +50,94 @@ def test_create_environment_with_repeated_name(self, client):
     def create_environment(client, environment):
         response = client.post(ENDPOINT, json=environment)
         return response
+
+    def test_get_environment_platform_service_config_forbidden(self, client):
```

Comment:
I don't think this endpoint should be admin protected. Only the post, put and delete should be protected.
Getting this config will be necessary for the creation of data outputs

Reply:
I think you will not use this endpoint outside of the settings page.  Data output will handle it on BE side. So, let's keep it protected for now.

Reply:
Alright, easy enough to fix should we still need it later indeed